bench/irmin-pack/layers.ml

```diff
@@ -267,7 +267,31 @@ let run config =
     Fmt.epr "After freeze thread finished : ";
     FSHelper.print_size config.root)
 
-let main () ncommits ncycles depth clear no_freeze show_stats =
+type result = {
+  number_of_commits : int;
```

Comment:
I guess this is not very important to report on the benchmark outputs (as hopefully they should always stay the same? :-))

Reply:
That's a good point!